CMakeLists.txt

```diff
@@ -901,16 +901,8 @@ endif()
 #       - AES-XTS
 #       - Web server
 #       - Web client
-
-# RABBIT
-if(NOT WOLFSSL_RABBIT)
-    list(APPEND WOLFSSL_DEFINITIONS "-DNO_RABBIT")
-else()
-    list(APPEND WOLFSSL_DEFINITIONS "-DHAVE_RABBIT")
-endif()
-
-# TODO: - RC2
-#       - FIPS, again (there's more logic for FIPS after RABBIT in configure.ac)
+#       - RC2
```

Comment:
'TODO' lost in change

Reply:
hmmm....That diff is misleading.  This is what that whole comment ends up looking like: 

```
# TODO: - CMAC
#       - AES-XTS
#       - Web server
#       - Web client
#       - RC2
#       - FIPS, again (there's more logic for FIPS in configure.ac)
#       - Selftest
```